### PR TITLE
Create readme.md: install needed softwares first.

### PR DESCRIPTION
**Pull request by changrui**

### CODE REVIEW

File: vlib/pg/readme.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+to use this module, do the fellowing first:
```
Comment (alexesprit) [inline vlib/pg/readme.md]: The first word in the sentense should be capitalized.

File: vlib/pg/readme.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+* Feodra 31 *
+'''
```
Comment (alexesprit) [inline vlib/pg/readme.md]: Triple grave accent ` should be used instead.

**Comment by alexesprit** [comment]
Looks nice for a start. Please, fix typos (Feodra -> Fedora, fellowing -> following) @vitalyster requested, and fix stylistic issues (see my comments).

**Comment by lutherwenxu** [comment]
It may also need some other commands from other distros.

**Comment by RiordanIX** [comment]
I made changes based on the requests. I don't have write access and I don't think I should make another pull request, so I'll just paste the file here for someone else to add it:

---------------------------------------------------------------

Before you can use this module, you must first have PostgreSQL installed on
your system. To do this, find to your OS and perform the actions. 

**NOTE**: These instructions are meant only as a convenience. If your OS is not
listed or you need extra help, [go here](https://www.postgresql.org/download/).

### Fedora 31
```
sudo dnf install postgresql-server postgresql-contrib
sudo systemctl enable postgresql # to autostart on startup
sudo systemctl start  postgresql
```

### Debian 10/11
```
sudo apt-get install postgresql postgresql-client
sudo systemctl enable postgresql # to autostart on startup
sudo systemctl start  postgresql
```

### MacOSX (Homebrew)
```
brew install postgresql
brew services start postgresql
```

**Comment by medvednikov** [comment]
Thanks @RiordanIX

**Pull request merged**